libraries/lib-wave-track/WaveTrack.h

```diff
@@ -65,6 +68,20 @@ class WAVE_TRACK_API WaveChannelInterval final : public ChannelInterval {
 
    const WaveClip &GetClip() const { return mClip; }
    const Envelope &GetEnvelope() const { return mEnvelope; }
+   bool Intersects(double t0, double t1) const;
+   double Start() const;
+   double End() const;
```

Comment:
Just as `Start` and `End` are non-virtuals in `ChannelGroupInterval`, so should the be too in the base class `ChannelInterval`, I think.  But that can be fixed later

Reply:
Indeed, whenever there's the need, we do this.